lib/amountable/amount.rb

```diff
@@ -2,20 +2,6 @@
 
 module Amountable
   class Amount < ActiveRecord::Base
-    class_attribute :columns
-    self.columns = []
-
-    def self.column(name, sql_type = nil, default = nil, null = true)
-      type = "ActiveRecord::Type::#{sql_type.to_s.camelize}".constantize.new
-      columns << ActiveRecord::ConnectionAdapters::Column.new(name.to_s, default, type, null)
-    end
-
-    column :amountable_type, :string, nil, false
-    column :amountable_id, :integer, nil, false
-    column :value_currency, :string, nil, true
-    column :value_cents, :integer, 0, false
-    column :name, :string, nil, false
```

Comment:
I don't believe any of this is needed since this class went back to being an ActiveRecord model

Reply:
is there a table that these attributes and types can be inferred from?

Reply:
This should all be covered by https://github.com/instacart/amountable/blob/9bfa804da14abad5bc2adb4fd60441df169a4c4c/db/migrate/0_create_amounts.rb#L5-L12